llvm/lib/Target/X86/MCTargetDesc/X86AsmBackend.cpp

```diff
@@ -267,8 +268,8 @@ static bool isRIPRelative(const MCInst &MI, const MCInstrInfo &MCII) {
 }
 
 /// Check if the instruction is a prefix.
-static bool isPrefix(const MCInst &MI, const MCInstrInfo &MCII) {
-  return X86II::isPrefix(MCII.get(MI.getOpcode()).TSFlags);
+static bool isPrefix(unsigned Opcode, const MCInstrInfo &MCII) {
```

Comment:
Not understand why this is better

Reply:
I think it's for this line in `X86AsmBackend::canPadInst`

```
if (isPrefix(PrevInstOpcode, *MCII))
```